travis/push-to-cicd-status.sh

```diff
@@ -32,19 +33,18 @@ git_clone_repo() {
     git remote set-url origin "${GIT_REPO/:\/\//://$GIT_USER:$GIT_TOKEN@}"
 }
 
-
 add_artifacts() {
     cd /tmp/"${GIT_LOCAL_DIR}" || exit
     git pull --rebase origin ${GIT_BRANCH}
-    mkdir -p /tmp/"${GIT_LOCAL_DIR}"/docs/release_artifacts/"${RELEASE_TAG}"/"${IMAGE}" 2> /dev/null
-    curl "https://api.travis-ci.com/v3/job/${TRAVIS_JOB_ID}/log.txt" > /tmp/"${GIT_LOCAL_DIR}"/docs/release_artifacts/"${RELEASE_TAG}"/"${IMAGE}"/"${RELEASE_TAG}"-buildlog.txt
-    cp /tmp/sbom.txt /tmp/"${GIT_LOCAL_DIR}"/docs/release_artifacts/"${RELEASE_TAG}"/"${IMAGE}"/"${RELEASE_TAG}"-sbom.txt
-    cp /tmp/cve.txt /tmp/"${GIT_LOCAL_DIR}"/docs/release_artifacts/"${RELEASE_TAG}"/"${IMAGE}"/"${RELEASE_TAG}"-cve.txt
-    cp /tmp/cve-base.txt /tmp/"${GIT_LOCAL_DIR}"/docs/release_artifacts/"${RELEASE_TAG}"/"${IMAGE}"/"${RELEASE_TAG}"-cve-base.txt
+    mkdir -p /tmp/"${GIT_LOCAL_DIR}"/docs/release_artifacts/"${RELEASE_TAG}"/z/"${IMAGE}" 2> /dev/null
```

Comment:
do we need "${RELEASE_TAG}"/"${RELEASE_TAG}".z/ or just "${RELEASE_TAG}"/z/"

Reply:
no i am saying for example instead of having 4.0.3.1/z have something like 4.0.3.1/4.0.3.1.z no ?